web/models.py

```
@@ -2008,7 +2011,23 @@
+    class ExamType(models.TextChoices):
+        SESSION = "session", "Session Exam"
+        COURSE = "course", "Course Exam"
+        QUIZ = "quiz", "Quiz"
+
+    exam_type = models.CharField(
+        max_length=10,
+        choices=ExamType.choices,
+        default=ExamType.QUIZ,
+    )
+
     STATUS_CHOICES = [("draft", "Draft"), ("published", "Published"), ("private", "Private")]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)